static/js/settings_account.js

```diff
@@ -209,16 +209,64 @@ exports.set_up = function () {
     $("#show_api_key_box").hide();
     $("#api_key_button_box").show();
 
+    function show_reset_email_loader() {
+        loading.make_indicator($(".loading_indicator_spinner"), {text: 'Sending...'});
+    }
+
+    function hide_reset_email_loader() {
+        loading.destroy_indicator($(".loading_indicator_spinner"));
+    }
+
+    function send_password_reset(type) {
+        var email = people.my_current_email();
+        var data = {};
+
+        data.email = email;
+        channel.post({
+            url: '/accounts/password/reset/',
+            data: data,
+            complete: function () {
+                hide_reset_email_loader();
+                if (type === 'user_password_reset') {
+                    $('#resend_link, #email_text').show();
+                } else {
+                    $('#api_resend_link, #api_email_text').show();
+                }
+            },
+        });
+    }
+
     $('#api_key_button').click(function () {
         if (page_params.realm_password_auth_enabled !== false) {
             $("#get_api_key_box").show();
         } else {
             // Skip the password prompt step
             $("#get_api_key_box form").submit();
         }
+        $('#api_forgot_password').show();
+        $('#api_reset_password').hide();
+        $('#api_resend_link').hide();
+        $('#api_email_text').hide();
```

Comment:
I'm always not sure about using hide and show on many selectors, it would be great if possible we can use some class and apply hide and show on that.

Reply:
I tried using class but it didn't work out

Reply:
Can you please elaborate on how the class didn't work out, I mean what's the problem with that approach, and if doesn't work out it's fine too.

Reply:
can I use seperate class for every selector?